OrbitClientData/ModuleData.cpp

```diff
@@ -8,14 +8,30 @@
 #include "OrbitClientData/FunctionUtils.h"
 #include "absl/synchronization/mutex.h"
 #include "capture_data.pb.h"
+#include "module.pb.h"
 
 using orbit_client_protos::FunctionInfo;
+using orbit_grpc_protos::ModuleInfo;
 
 bool ModuleData::is_loaded() const {
   absl::MutexLock lock(&mutex_);
   return is_loaded_;
 }
 
+void ModuleData::Update(ModuleInfo info) {
```

Comment:
Would it be better to call this `UpdateIfChanged` as the update doesn't necessarily happen?

Reply:
done